src/scripts/start-stop-status

```diff
@@ -31,7 +31,7 @@ fi
 start_daemon() {
     local ts=$(date --iso-8601=second)
     echo "${ts} Starting ${SERVICE_NAME} with: ${SERVICE_COMMAND}" >${LOG_FILE}
-    STATE_DIRECTORY=${PKGVAR} ${SERVICE_COMMAND} >>${LOG_FILE} 2>&1 &
+    STATE_DIRECTORY=${PKGVAR} ${SERVICE_COMMAND} >/dev/null 2>&1 &
```

Comment:
can you get pipe through `sed '1,200p;d'` in here (per tip from @creachadair) so we only print the first 200 lines and then discard the rest? then we still see start-up messages.

Reply:
Still working on this, something isn't right. No output from tailscaled is being written to the file.

```
dgentry@dsm7:/var$ cat  /var/packages/Tailscale/var/tailscaled.stdout.log
2023-01-19T18:18:55-08:00 Starting tailscale with: /volume1/@appstore/Tailscale/bin/tailscaled --state=/volume1/@appdata/Tailscale/tailscaled.state --socket=/volume1/@appdata/Tailscale/tailscaled.sock --port=41641 --tun=userspace-networking
dgentry@dsm7:/var$ wc -l  /var/packages/Tailscale/var/tailscaled.stdout.log
1 /var/packages/Tailscale/var/tailscaled.stdout.log
dgentry@dsm7:/var$
```

It is GNU sed, not busybox or something weird, and `sed '1,200p;d'` on a text file does print the first 200 lines.
```
dgentry@dsm7:/var$ /bin/sed --version
/bin/sed (GNU sed) 4.5
Copyright (C) 2018 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <https://gnu.org/licenses/gpl.html>.
```